test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1222,6 +1121,28 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion Content Header Inclusion
 
+    #region Client Certificate Authentication
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
+        $uri = 'https://localhost:8083/'
```

Comment:
@iSazonov  It's a possibility. Should I add that now or include it in future PRs when I start adding features to the new listener and moving tests over? HTTPS might be tricky if we need to run multiple.